packages/web/src/pages/saved-page/components/desktop/SavedPage.module.css

```diff
@@ -115,3 +115,13 @@
 .createPlaylistIcon circle {
   fill: var(--neutral-light-2);
 }
+
+.spinner {
+  height: 40px;
+  width: 40px;
+  margin: 0 auto;
+}
+
+.spinner g path {
+  stroke: var(--secondary) !important;
```

Comment:
Is the `!important` actually needed? I just styled a loading spinner in another PR and it took the stroke color without it.
Also we should really encapsulate the color/sizes for LoadingSpinner into the component the same way we do for `Icon`